tools/wpt/testharnessreport.js

```diff
@@ -8,6 +8,10 @@ window.add_result_callback(({ message, name, stack, status }) => {
   );
 });
 
+// TODO(kt3k): Enable the below hook and timers test when #10445 is fixed
+// ref: https://github.com/denoland/deno/issues/10445
+/*
 window.add_completion_callback((tests, harnessStatus) => {
   Deno.exit(0);
 });
+*/
```

Comment:
What if you call `Deno.exit` inside a `queueMicrotask`?

Reply:
That didn't make difference (still causes swallowing of uncaught error). If I use setTimeout, then it works for the file-constructor test etc, but it makes timers tests flaky.